libs/blocks/global-navigation/global-navigation.css

```diff
@@ -418,6 +382,14 @@ header .app-launcher {
 
 /* END APP LAUNCHER */
 
+.feds-profile {
```

Comment:
This should be moved up. Right now it's in the `/* TODO: Old styles, to be refactored */` section. These styles can go on L186 with a `/* Profile */` comment, for consistency purposes.

Reply:
Good catch, changed 👍